src/controllers/projects.js

```diff
@@ -1,42 +1,104 @@
-// import Project from '../models/Project';
-// import fs from 'fs';
+import Project from '../models/Project';
 
 const projects = [];
 
-const create = (req, res) => {
-  const projectData = req.body;
-  if (projectData.id && projectData.name && projectData.status && projectData.description
-    && projectData.employees && projectData.startDate && projectData.endDate) {
-    projects.push(projectData);
-    // fs.writeFile('src/data/projects.json', JSON.stringify(projects), (err) => {
-    //   if (err) {
-    //     res.send(err);
-    //   } else {
-    //     res.send('Project create');
-    //   }
-    // });
-  } else {
-    res.send('project not created');
+const create = async (req, res) => {
+  try {
+    const project = new Project({
+      name: req.body.name,
+      status: req.body.status,
+      description: req.body.description,
+      employees: req.body.employees,
+      startDate: req.body.startDate,
+      endDate: req.body.endDate,
+    });
```

Comment:
Para simplificar este codigo, se puede hacer lo siguiente:

```suggestion
    const project = new Project(req.body);
```

De esta manera, le pasas el objeto completo  ya que contiene todas las keys y values que el modelo y el objeto